src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -167,11 +168,16 @@ class MulticopterAttitudeControl
 	perf_counter_t	_loop_perf;			/**< loop performance counter */
 	perf_counter_t	_controller_latency_perf;
 
-	math::Vector<3>		_rates_prev;	/**< angular rates on previous step */
-	math::Vector<3>		_rates_sp_prev; /**< previous rates setpoint */
+	math::LowPassFilter2p _lp_filters_d[3];                      /**< low-pass filters for D-term (roll, pitch & yaw) */
+	static constexpr const float initial_update_rate_hz = 250.f; /**< loop update rate used for initialization */
```

Comment:
I think we're soon going to need a mechanism to set this system wide (single param or programmatically).

Reply:
This is only for initialization. A module needs to be able to determine this on its own and at runtime.